### PR TITLE
feat: prodexa API integration

### PR DESCRIPTION
**Pull request by EugeneGoroschenya**
https://github.com/prodexa/pxm/issues/4312

See also:
* https://github.com/prodexa/pxm/pull/4407
* https://github.com/prodexa/pxm/pull/4419

### CODE REVIEW

File: src/lib/services/prodexa/prodexa-utils.ts [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  }
+}
+
+
+export const mapProdexajsProduct = (p: any) => {
+  if (p) {
+
+    let img = ''
+    if (p?.docAssociations?.length > 0) {
+     img = "http://localhost:8082/pxm/workarea/" + p?.docAssociations[0].docAssociation_path
```
Comment (EugeneGoroschenya) [inline src/lib/services/prodexa/prodexa-utils.ts:26]: Don't add prefix "http://localhost:8082/pxm/workarea/" here

File: src/lib/services/prodexa/prodexa-utils.ts [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    }
+
+    let images = []
+    if (p?.docAssociations?.length > 0) {
+      images = p?.docAssociations.map((doc) => 'http://localhost:8082/pxm/workarea/'  + doc.path)
```
Comment (EugeneGoroschenya) [inline src/lib/services/prodexa/prodexa-utils.ts:31]: Don't add prefix "http://localhost:8082/pxm/workarea/" here

File: src/lib/services/prodexa/prodexa-utils.ts [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+      name: p.manufacturerId,
+      slug: p.manufacturerId,
+
+    }
+
+
+    const prod: Product = {
+      _id: p.catalogId + '___' + p.productId,
+      id:  p.catalogId + '___' + p.productId,
+      slug: p.catalogId + '___' + p.productId,
```
Comment (EugeneGoroschenya) [inline src/lib/services/prodexa/prodexa-utils.ts:67]: Use proper way for id and slug to point product.

File: src/lib/components/Image/LazyImg.svelte [hunk 4]
```diff
@@ -157,5 +157,14 @@ if (extension === 'svg') {
...
-	{/if}
+
+  {:else if IMAGE_CDN_PROVIDER === 'prodexa'}
+    <img
+    {alt}
+    src="{src}"
+    loading="lazy"
+    height="{+h}"
+    width="{+w}"
+    class="aspect-[{aspect_ratio?.split(':')[0]}/{aspect_ratio?.split(':')[1]}] lazy {clazz}" />
```
Comment (EugeneGoroschenya) [inline src/lib/components/Image/LazyImg.svelte:168]: Perhaps it makes sense to make it generic for any IMAGE_CDN_PROVIDER

File: src/lib/services/shopify/address-service.ts
Comment (EugeneGoroschenya) [inline src/lib/services/shopify/address-service.ts:1]: Revert back shopify adapter

File: src/lib/components/Breadcrumb.svelte [hunk 5]
```diff
@@ -55,17 +55,17 @@ export { clazz as class }
...
+			<!--	<svg-->
+			<!--		xmlns="http://www.w3.org/2000/svg"-->
+			<!--		class="h-5 w-5 px-1 shrink-0"-->
+			<!--		fill="none"-->
+			<!--		viewBox="0 0 24 24"-->
+			<!--		stroke="currentColor"-->
+			<!--		stroke-width="2">-->
+			<!--		<path stroke-linecap="round" stroke-linejoin="round" d="M9 5l7 7-7 7"></path>-->
+			<!--	</svg>-->
+			<!--{/if}-->
```
Comment (EugeneGoroschenya) [inline src/lib/components/Breadcrumb.svelte:68]: please have a look at

image

why should we comment it?

Comment (OlegGoloubitski) [reply src/lib/components/Breadcrumb.svelte]: agree to revert, even if it looks confusing

**Pull request merged**